src/Action.jsx

```diff
@@ -46,26 +46,36 @@ function Action(props) {
         styles.action,
       ])}
     >
-      <img
-        alt={action.user.name}
-        src={action.user.thumbSrc}
-        className={styles.thumb}
-      />
-      <div className={styles.info}>
-        <div className={styles.infoHead}>
-          <div className={styles.name}>{action.user.name}</div>
-          {!action.thread.is_canonical &&
-            [
-              <div className={styles.convIn}>in</div>,
-              <div className={styles.convName}>{action.thread.name}</div>,
-            ]
-          }
-        </div>
-        <div className={styles.date}>{moment(action.endDate).fromNow()}</div>
-        <div className={styles.typings}>
-          {typings}
+      <a
+        className={cx([
+          styles.actionLink,
```

Comment:
No need to use `cx` here since we only have one className. `className={styles.actionLink}`